src/actors/FilterBarActor.js

```diff
@@ -0,0 +1,88 @@
+import * as SharedUtils from '../utils/SharedUtils';
+
+export class FilterBarActor {
```

Comment:
If a FilterBarActor is being passed right down the component tree, isn't it just a giant collection of callbacks? Isn't this precisely what Flux is trying to avoid?